ini/parse.ts

```diff
@@ -1,9 +1,63 @@
 // Copyright 2018-2025 the Deno authors. MIT license.
 // This module is browser compatible.
 
-import { IniMap, type ReviverFunction } from "./_ini_map.ts";
+import type { ReviverFunction } from "./_ini_map.ts";
 export type { ReviverFunction };
 
+const SECTION_REGEXP = /^\[(?<name>.+)]$/;
+const KEY_VALUE_REGEXP = /^\s*(?<key>.*?)\s*=\s*(?<value>.*?)\s*$/;
```

Comment:
The leading and trailing whitespaces seem redundant to me as we use this regexp to trimmed lines

Reply:
You are right, removed for now. I think I will try to optimize this in the spirit of https://github.com/denoland/std/pull/5751 at a later point.